test_vm/bootstrap.sh

```diff
@@ -0,0 +1,8 @@
+# Create new physical volume.
+pvcreate /dev/sdb
+# Add the physical volume to the existing volume group.
+vgextend centos /dev/sdb
+# Add the physical volume to the logical volume.
+lvextend /dev/centos/root /dev/sdb
+# Resize the file system to use the new space available in the logical volume..
+xfs_growfs /dev/centos/root
```

Comment:
Hmmm...I still need to add some more error trapping here (test the exit status of all of the commands).